spec/features/cards_spec.rb

```diff
@@ -21,6 +21,26 @@
       end
     end
 
+    describe 'card in pack' do
+      let!(:user) { create(:user, current_pack_id: 1) }
+      let!(:pack) { create(:pack, current_user_id: 1) }
```

Comment:
use relative links in tests.
Create user first
after it create pack
`let!(:pack) { create(:pack, current_user_id: user.id) }`